src/js/login.js

```diff
@@ -43,14 +43,26 @@ pass.addEventListener("click", ()=>{
     pass.innerHTML = ($("#cPass").toggle().is(":visible"))?("Click here to hide change password form"):("Click here to change password");
 });
 
+var password_validation = () => {
+    if (confirmPassword.value === newPassword.value && newPassword.value.length >6 && newPassword.value.length<64 && cpassword.value.length>0) {
```

Comment:
Change it to `newPassword.value.length >=6 && newPassword.value.length<=64`

Reply:
@neeraj3029 Done the required changes.